pkg/login/social/github_oauth.go

```diff
@@ -124,27 +131,27 @@ func (s *SocialGithub) FetchTeamMemberships(client *http.Client) ([]int, error)
 			return nil, fmt.Errorf("Error getting team memberships: %s", err)
 		}
 
-		var records []Record
+		var records []GithubTeam
 
 		err = json.Unmarshal(response.Body, &records)
 		if err != nil {
 			return nil, fmt.Errorf("Error getting team memberships: %s", err)
 		}
 
 		newRecords := len(records)
-		existingRecords := len(ids)
-		tempIds := make([]int, (newRecords + existingRecords))
-		copy(tempIds, ids)
-		ids = tempIds
+		existingRecords := len(teams)
+		tempTeams := make([]GithubTeam, (newRecords + existingRecords))
+		copy(tempTeams, teams)
+		teams = tempTeams
 
 		for i, record := range records {
-			ids[i] = record.Id
+			teams[i] = record
```

Comment:
This seems like the current version is as `i` will start from `0` every time.  We can avoid all the copying etc if we just say:
```
var teams []GithubTeam
...
for _, record := range records {
  teams = append(teams, record)
}
```

Reply:
Using append here simplifies the code a lot.